--- example_toptagger.orig.cc	2014-12-07 22:44:01.261902682 +0800
+++ example_toptagger.cc	2014-12-07 22:44:13.481902957 +0800
@@ -22,7 +22,7 @@
 
 #include "fastjet/PseudoJet.hh"
 #include "fastjet/ClusterSequence.hh"
-#include "fastjet/CDFJetCluPlugin.hh"
+// #include "fastjet/CDFJetCluPlugin.hh"
 #include "fastjet/internal/BasicRandom.hh"
 
 using namespace std;
@@ -65,7 +65,7 @@
     {
       double topmass=172.3;
       double wmass=80.4;
-      HEPTopTagger::HEPTopTagger cm_toptag(clust_seq,jets[ijet],topmass,wmass);
+      HEPTopTagger cm_toptag(clust_seq,jets[ijet],topmass,wmass);
       cm_toptag.set_top_range(150.,200.);
       cout<< "========= Top Tagger ============" << endl;
       cm_toptag.run_tagger();
